test-runner-frontend/src/Components/IdCell.js

```diff
@@ -23,7 +23,7 @@ export default function IdCell({ value, hasCheckbox, onChange, status, row }) {
     iconColor = 'green';
     IconComponent = CheckCircleOutlinedIcon;
     textDecoration = 'underline';
-    routingReference = '/report';
+    routingReference = `/report/${row.id}`;
```

Comment:
as much as this is correct, i would recommend we use `generatePath`: https://reactrouter.com/en/main/utils/generate-path

Reply:
Done.